src/components/Header.js

```diff
@@ -1,21 +1,81 @@
-import React from 'react';
-import SupportedCurrencies from './SupportedCurrencies';
-import { BrowserRouter as Router, Route, Link, BrowserRouter } from "react-router-dom";
+/*  Added the new Button component.  The props are passed in for the content of the button and its
+		onClick method.
+*/
 
+import React from 'react';
+import Radium from 'radium';
+import Button from './Button';
+import RadiumVars from '../RadiumVariables';
 
 const Header = (props) => {
 	return (
-    <div className="main__flex">
-      <div className="header">
-        <h1 className="header__title" onClick={props.handleHideCurrencyList}>CryptoTraq</h1>
+    <div style={styles.header__flex}>
+      <div style={[styles.header]}>
+        <h1 style={styles.header__title} onClick={props.handleHideCurrencyList}>CryptoTraq</h1>
       </div>
-      <div className="supportedCurr">
-        <button className="btn btn--light" onClick={props.handleShowCurrencyList}>
-          Supported currencies
-        </button>
+      <div style={styles.header__button_box}>
+        <Button onClick={props.handleShowCurrencyList} buttonContent={"Supported currencies"}/>
```

Comment:
`Button` component is in use here as well.  It has its `onClick` method and its `buttonContent` passed in via props.